static/templates/settings/realm_domains_table.hbs

```diff
@@ -0,0 +1,24 @@
+<table class="table table-condensed table-bordered wrapped-table" id="realm_domains_table">
+    <thead>
+        <th>{{t "Domain" }}</th>
+        <th>{{t "Allow subdomains"}}</th>
+        <th>{{t "Action" }}</th>
+    </thead>
+    <tbody>
+    </tbody>
+    {{#if is_admin }}
```

Comment:
Shouldn't we just hide the whole table for non-admins, instead of showing a blank table?

Reply:
We do not show the whole of organization permissions for normal users. I think it's redundant here.(I renamed file from realm_domains_modal.hbs and remove little code, it was from that page. Thought it's probably needed)
Can I remove that?

Reply:
Lets remove this then. We should strive to keep templates free of logic where possible.

Reply:
Removed.